File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1893.rb
while linha = gets
        
    ls = linha.split(' ')
    
    a = (ls[0]).to_i
    b = (ls[1]).to_i
    
    if(b >= 0 and b <= 2) then puts("nova")
    elsif(b >= 97 and b <= 100) then puts("cheia")
    else
        if(a > b) then puts("minguante")
        else puts("crescente") end
    end

end

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1893.cs
using System;

class URI
{
    static void Main() {
        while(true) {
            string linha = Console.ReadLine();
            if (string.IsNullOrEmpty(linha))
                break;
            string[] ls = linha.Split(' ');

            int a = int.Parse(ls[0]), b = int.Parse(ls[1]);

            if(b >= 0 && b <= 2) Console.Write("nova\n");
            else if(b >= 97 && b <= 100) Console.Write("cheia\n");
            else{
                if(a > b) Console.Write("minguante\n");
                else Console.Write("crescente\n");
            }
            
        }
    }
}
